core/src/components/cat-select/cat-select.tsx

```diff
@@ -149,7 +149,8 @@ export class CatSelect {
   }
 
   @Watch('value')
-  setValueHandler(value?: any) { // eslint-disable-line @typescript-eslint/no-explicit-any
+  setValueHandler(value?: any) {
+    // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
to pass the check of any (also prettier check), you can use eslint-disable-next-line above the method